src/form/Input/InputStyle.ts

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { themeColor, themeSpacing } from '@amsterdam/asc-ui';
+import { INPUT_SIZE } from '../../shared/constants';
+
+// @todo style [required] when design is available
+
+export const InputStyle = styled.input`
```

Comment:
Ik zou hier toch de voorkeur geven aan het gebruik van Input uit ASC. Hoe meer we opnieuw het wiel uit gaan vinden hoe lastiger je het jezelf gaat maken.